package.json

```diff
@@ -64,6 +64,7 @@
     "@types/estree": "^1.0.0",
     "@types/jest": "^26.0.23",
     "@types/node": "^17.0.0",
+    "@types/sinon": "^10.0.13",
```

Comment:
These new dependencies should not be added to the package.json of the root folder but to the taquito-bacon-wallet one.

Reply:
moved "@types/sinon": "^10.0.13", to taquito-bacon-wallet package json